bundle.md

```diff
@@ -20,3 +20,6 @@ This directory MUST be referenced from within the `config.json` file.
 
 While these artifacts MUST all be present in a single directory on the local filesystem, that directory itself is not part of the bundle.
 In other words, a tar archive of a *bundle* will have these artifacts at the root of the archive, not nested within a top-level directory.
+
+
+[macos_bundle]: http://en.wikipedia.org/wiki/Bundle_%28OS_X%29
```

Comment:
This redirects to https://en.wikipedia.org/wiki/Bundle_%28macOS%29.  Since you've updated the link text above, I suggest updating the URL here as well.